test/test.js

```diff
@@ -1,22 +1,106 @@
-const request = require('supertest');
+const request = require("supertest");
+const assert = require("assert");
 
-describe('loading express', () => {
+describe("loading express", () => {
   let server;
   before(() => {
-    server = require('../src/index');
+    server = require("../src/index");
   });
-  after((done) => {
+  after(done => {
     server.close(done);
   });
 
   // ? NOTE: Only _one_ server instance for all tests right now.
   // ?      If this needs to be changed, it can.
 
-  it('responds to /', (done) => {
-    request(server).get('/api').expect(200, done);
+  it("responds to /", done => {
+    request(server)
+      .get("/api")
+      .expect(200, done);
   });
-  it('404 bad requests', (done) => {
-    request(server).get('/bad/request').expect(404);
-    request(server).get('/api/customer-bad-api').expect(404, done);
+  it("404 bad requests", done => {
+    request(server)
+      .get("/bad/request")
+      .expect(404);
+    request(server)
+      .get("/api/customer-bad-api")
+      .expect(404, done);
+  });
+  it("good request (no timezone)", done => {
+    request(server)
+      .get("/api/customer-io?curr=6&max=10")
+      .set("Accept", "application/json")
+      .expect("Content-Type", /json/)
+      .expect(200, done);
+  });
+  it("good request (timezone)", done => {
+    request(server)
+      .get("/api/customer-io?curr=6&max=10&timezone=EDT")
+      .set("Accept", "application/json")
+      .expect("Content-Type", /json/)
+      .expect(function(res) {
```

Comment:
I'm not sure about this one: is this supposed to be an .expect callback or a .then? Looks like you are just logging the result.